node/node_handler.go

```diff
@@ -791,6 +791,9 @@ func (node *Node) bootstrapConsensus() {
 					Msg("No peers, continue")
 				continue
 			}
+			utils.Logger().Info().
+				Int("numPeersNow", numPeersNow).
+				Msg("No peers, continue")
```

Comment:
1. Print this only if `numPeersNow` has increased from the last tick, so as to prevent infinite loop of log messages if peer discovery fails.
2. The message is misleading.  How about moving this log line after the `MinPeers` check below, and saying "still waiting for more peers"?
3. It would help to log both `numPeersNow` and `minPeers` so as to make the target number explicit and aid comparison.

Reply:
Sorry. too rushed to add the change. Fixed.